Source/Particles/PhotonParticleContainer.cpp

```diff
@@ -21,12 +21,27 @@ using namespace amrex;
 PhotonParticleContainer::PhotonParticleContainer (AmrCore* amr_core, int ispecies,
                                                   const std::string& name)
     : PhysicalParticleContainer(amr_core, ispecies, name)
-{}
+{
+
+    ParmParse pp(species_name);
+
+#ifdef WARPX_QED
```

Comment:
An alternative here could be:
if do_qed_breit_wheeler=1, then QED processes are enabled for a given species (i.e. do_qed=1)

Reply:
I would remove the compilation flag. I don't think that compilation with QED effects is significantly longer than the compilation without QED effects. However, we could do tests and we could discuss that in detail. However, for the moment, I would keep the compilation flag, since QED effects require the picsar library to be on a specific branch (QED)

Reply:
Update: I've done a quick test. 
time make DIM=2                       --> 5m58,979s
time make DIM=2 QED=TRUE --> 6m29,229s
So, there seems to be  a significant difference (~ 8 %), though not huge.

Reply:
Ok, let's keep all tests.
And thanks for the timers!